src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
-  margin: 0;
-}
+  html {
+  color: #000;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  background: gray;
+  font-family: Roboto, sans-serif;
+  }
+
+  body, ul, li {
+    margin: 0;
+  }
+
+  .container {
```

Comment:
`color` by default is black, no?
`font-style` by the default is normal
![image](https://github.com/mate-academy/layout_moyo-header/assets/91411692/316bffa1-fb30-412c-8ea2-72f3c007d103)

```suggestion
html {
  font-weight: 500;
  line-height: normal;
  text-transform: uppercase;
  background: gray;
  font-family: Roboto, sans-serif;
}

 body, ul, li {
  margin: 0;
}

.container {
```